### PR TITLE
Vue3 activity and i18next

### PR DESCRIPTION
**Pull request by UtkarshSiddhpura**
This PR includes:

- Updated Vue template to use Vue 3 and i18next.
- Updated tutorial docs to reflect the above changes.

### CODE REVIEW

**Review by Vishalk91-4 — COMMENTED** [review]
@UtkarshSiddhpura, I found that there were like 2-3 files based on chnaging quotes for most of their content, I guess a formatting issue of your IDE

Rest the code is changed completely
I'll run the steps to make sure if it's working fine or not

File: activities/ActivityTemplate/VueJS/js/components/SugarDevice.js [hunk 1]
```diff
@@ -58,33 +65,39 @@ Vue.component('sugar-device', {
...
-				{ s: 'Linux', r: /(Linux|X11(?!.*CrOS))/ },
-				{ s: 'iOS', r: /(iPhone|iPad|iPod)/ },
-				{ s: 'Mac OS X', r: /Mac OS X/ },
-				{ s: 'Mac OS', r: /(MacPPC|MacIntel|Mac_PowerPC|Macintosh)/ },
-				{ s: 'QNX', r: /QNX/ },
-				{ s: 'UNIX', r: /UNIX/ },
-				{ s: 'BeOS', r: /BeOS/ },
-				{ s: 'OS/2', r: /OS\/2/ },
-				{ s: 'Search Bot', r: /(nuhk|Googlebot|Yammybot|Openbot|Slurp|MSNBot|Ask Jeeves\/Teoma|ia_archiver)/ }
+				{ s: "Windows 10", r: /(Windows 10.0|Windows NT 10.0)/ },
```
[the remark this answers is not in the document]
Comment (UtkarshSiddhpura) [reply activities/ActivityTemplate/VueJS/js/components/SugarDevice.js:68]: yea it is for maintaining consistent code style, as it's not maintained in vue2 components.

File: docs/tutorial/VueJS/step6.md [hunk 2]
```diff
@@ -154,7 +162,6 @@ data: {
 	...
```
Comment (Vishalk91-4) [inline docs/tutorial/VueJS/step6.md:162]: Changing this also to 

data : function() {
        return {
                currentenv: null,
                SugarL10n: null,
                SugarPresence: null,
                ...

File: docs/tutorial/VueJS/step9.md [hunk 3]
```diff
@@ -90,6 +90,14 @@ So, let's create a new Javascript file `lib/tutorial.js` to handle everything re
...
+```js
+// Register the component
+const app = Vue.createApp({
+	components: {
+		"sugar-tutorial": SugarTutorial,
+		...
+	},
+```
 
 ## Localize the tutorial
```
Comment (Vishalk91-4) [inline docs/tutorial/VueJS/step9.md:103]: Changing this to

locales/`<lang>`.json

`<lang>`.json

```
	"TutoPrev": "Prev",
	"TutoNext": "Next",
	"TutoExplainTitle": "Pawn Activity",
	"TutoExplainContent": "Welcome into the Pawn activity. This activity is an activity to test Sugarizer development.",
	"TutoAddTitle": "Add pawn",
	"TutoAddContent": "Click here to add one to three pawns on the board.",
	"TutoBackgroundTitle": "Change background",
	"TutoBackgroundContent": "Click here to choose a new background for the board.",
```

**Comment by Vishalk91-4** [comment]
@llaske 

Checked, by making the pawn activity
Everything works fine

File: docs/tutorial/VueJS/step6.md (hunk 2 above)
Comment (UtkarshSiddhpura) [reply docs/tutorial/VueJS/step6.md]: done, thanks

File: docs/tutorial/VueJS/step9.md (hunk 3 above)
Comment (UtkarshSiddhpura) [reply docs/tutorial/VueJS/step9.md:103]: done, thanks!

**Comment by llaske** [comment]
Good job. Thanks @UtkarshSiddhpura

**Pull request merged**